packages/web-components/src/toolbar/toolbar.spec.md

```diff
@@ -0,0 +1,88 @@
+# Toolbar
```

Comment:
There is a missing README.md for the Toolbar. It will contain helpful topics, such as a prop mapping between React v9 and Web Components v3. See other Web Component directories README.md files for examples.

Reply:
Addressed in new PR.